chapters/en/Unit 1 - Fundamentals/Chapter 1 - Fundamentals of Computer Vision/defination.mdx

```diff
@@ -1,36 +1,16 @@
-videos, and other forms of visual data.
 
-## Deep learning and the computer vision renaissance
-
-The evolution of computer vision has been marked by a series of incremental advancements in
-and across its interdisciplinary fields, where each step forward gave rise to breakthrough algorithms,
-hardware, and data, giving it more power and flexibility. One such history was the jump to the widespread
-use of deep learning methods.
+Computer vision is the science and technology of making machines see. It involves the development of theoretical and algorithmic methods to acquire, process, analyze, and understand visual data, and to use this information to produce meaningful representations, descriptions, and interpretations of the world. (*Forsyth & Ponce, Computer Vision: A Modern Approach*)
 
-Initially, to extract and learn the information in an image, you extract features through image-preprocessing
-techniques (chapter 3). Once you have a group of features describing your image, you use a classical
-machine learning algorithm for your dataset of features. It is a strategy that already simplifies things
-from the hard-coded rules, but it still relies on domain knowledge and exhaustive feature engineering.
-The competing strategy arises when deep learning methods and large datasets meet.
-Deep learning (DL) allows machines to learn complex features from the raw data automatically. This paradigm
-shift allowed us to build more adaptive and sophisticated models causing a renaissance in the field.
+## Deep learning and the computer vision renaissance
 
-We wanted to emphasize here that the compute vision community is made up of researchers,
-engineers, data scientists, and passionate hobbyists across the globe coming from a vast array
-of disciplines. This community maintains the Open-source initiates, leads community-driven projects,
-and democratizes access to cutting-edge tools and technologies.
+The evolution of computer vision has been marked by a series of incremental advancements in and across its interdisciplinary fields, where each step forward gave rise to breakthrough algorithms, hardware, and data, giving it more power and flexibility. One such leap was the jump to the widespread use of deep learning methods.
 
+Initially, to extract and learn information in an image, you extract features through image-preprocessing techniques (chapter 3). Once you have a group of features describing your image, you use a classical machine learning algorithm on your dataset of features. It is a strategy that already simplifies things from the hard-coded rules, but it still relies on domain knowledge and exhaustive feature engineering. A more state-of-the-art approach arises when deep learning methods and large datasets meet. Deep learning (DL) allows machines to automatically learn complex features from the raw data. This paradigm shift allowed us to build more adaptive and sophisticated models, causing a renaissance in the field.
```

Comment:
We tried to emphasize Open source and its role here but somehow not fitting properly so i removed it entirely with few typo changes.

Reply:
I think it would be nice if we tried a do over. I mean, we can write a paragraph about the history and just have one line that goes: " compute vision community is made up of researchers,
engineers, data scientists, and passionate hobbyists across the globe coming from a vast array
of disciplines".

Reply:
I'll redo and get back to you